### ISSUE TITLE
log.exception does not show the full stack trace of the problem

### ISSUE THREAD

**Issue by FesonX** [yoobool/flask-state#18]
**Describe the bug**
When I use `log.exception`, the Flask-State does not log trace as expected. <br>
It just show the error msg like use `log.error`

**To Reproduce**
1. Run the `examples/app.py` without change the example Redis config
2. Change the `services/host_status` in  line 94 from `log.warning(t)` to `log.exception(t)`
3. When trigger the record events, timeout error will be raised

**Expected behavior**
log.exception will print full stack trace of the error

**Actual behavior**
Simply log `imeout connecting to server` and with confuse code line like the following output:

```
FLASK_APP = examples/app.py
FLASK_ENV = development
FLASK_DEBUG = 1

flask-state/env/bin/python -m flask run
 * Serving Flask app "examples/app.py" (lazy loading)
 * Environment: development
 * Debug mode: on
 * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
 * Restarting with stat
2020-11-22 23:11:20,821 | INFO | manager:record_timer:83 - Acquired file lock successfully. process ID: 29123
 * Debugger is active!
 * Debugger PIN: 161-836-542
127.0.0.1 - - [22/Nov/2020 23:11:41] "GET / HTTP/1.1" 200 -
2020-11-22 23:12:01,012 | ERROR | __init__:exception:1413 - Timeout connecting to server
2020-11-22 23:12:01,025 | INFO | host_status:create_host_status:52 - Insert status
```

**Additional context**
Commit ID  is `11c2ea4b29c40b3fa10cb85b38986eeca0b93ce3`

**Comment by aaajiulong** [comment]
This problem is because Logger.exception is a convenience method to Logger.error, so the output code line will point to the location of Logger.error. We will replace the exception method with error.

**Comment by aaajiulong** [comment]
Fixed by [```c570d3f4```](https://github.com/yoobool/flask-state/commit/c570d3f4cf7afb7b4807df95a0c26342899fce47).
